core/vm/actor/builtin/types/miner/quantize.cpp

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "vm/actor/builtin/types/miner/quantize.hpp"
+
+namespace fc::vm::actor::builtin::types::miner {
+  QuantSpec::QuantSpec(ChainEpoch unit, ChainEpoch offset)
+      : unit(unit), offset(offset) {}
+
+  ChainEpoch QuantSpec::quantizeUp(ChainEpoch e) const {
+    const auto rounded_offset = offset % unit;
```

Comment:
Should `bigmod` be used instead of `%`?

Reply:
lotus uses integer `%` and `/` for this method, chain epoch is not big

Reply:
No, %